metadata-db-app/src/cmr/metadata_db/api/generic_documents.clj

```diff
@@ -6,20 +6,21 @@
 
 (defn- create-generic-document
   "Create a Generic Document"
-  [context params provider-id body]
-  (let [result (gen-doc/insert-generic-document context params provider-id body)]
+  [context params provider-id native-id body]
+  (let [result (gen-doc/insert-generic-document context params provider-id native-id body)]
     {:status 200 :body result}))
 
 (defn- read-generic-document
   "Read a Generic Document"
-  [context params provider-id concept-id]
-  (let [result (gen-doc/read-generic-document context params provider-id concept-id)]
+  [context params provider-id native-id]
+  (let [result (gen-doc/read-generic-document context params provider-id native-id)]
     {:status 200 :body result}))
 
 (defn- update-generic-document
   "Update a Generic Document"
-  [context params provider-id concept-id body]
-  (let [result (gen-doc/update-generic-document context params provider-id concept-id body)]
+  [context params provider-id native-id body]
+  (println "here with native-id" native-id "about to do an update...")
```

Comment:
This println statement should be removed.

Reply:
missed that one